installation_tips/README.md

```diff
@@ -64,3 +62,10 @@ This script tests the following:
   * running herdingspikes (not on windows)
   * opening the spikeinterface-gui
   * exporting to Phy
+
+
+## Installing before release
+
+Some tools in the spikeinteface ecosystem are getting regular bug fixes (spikeinterface, spikeinterface-gui, probeinterface, python-neo, sortingview).
+We are making releases 2 to 4 times a year. In between releases if you want to install from source you can use the `full_spikeinterface_environement_rolling_updates.yml` file to create the environement. This will install packages of the ecosystem from source.
```

Comment:
```suggestion
We are making releases 2 to 4 times a year. In between releases if you want to install from source you can use the `full_spikeinterface_environment_rolling_updates.yml` file to create the environment. This will install the packages of the ecosystem from source.
```

You misspelled the yaml name if you want to fix that overall. I fixed the typo here.